itemizedReceipt.js

```diff
@@ -6,13 +6,67 @@
 
 
 // Check
-logReceipt(
-  { descr: 'Burrito', price: 5.99 },
-  { descr: 'Chips & Salsa', price: 2.99 },
-  { descr: 'Sprite', price: 1.99 }
-);
+let logReceipt =[
+  { descr: 'Burrito', 
+    price: 5.99,
+    type: 'Food'
+  },
+  { descr: 'Chips & Salsa', 
+    price: 2.99,
+    type: 'Food'
+  },
+  { descr: 'Sprite', 
+    price: 1.99,
+    type:'Drink',
+    oz: 12,
+    sugar: true
+  }
+]
+  ;
 // should log something like:
 // Burrito - $5.99
 // Chips & Salsa - $2.99
 // Sprite - $1.99
 // Total - $10.97
+
+// source: https://dor.wa.gov/get-form-or-publication/publications-subject/tax-topics/seattle-s-sweetened-beverage-tax
+const seattleSugarTaxRate = .0175
+
+//source: http://www.salestaxstates.com/sales-tax-calculator-washington-seattle-98101
+const washingtonSalesTax = .065
+const seattleSaleTax = .0375
+const wholeSalesTax = washingtonSalesTax + seattleSaleTax
+
+function Receipt(){
```

Comment:
Remember the good naming conventions, variables and functions ought to always begin with a lowercase letter and use camelCase. By naming this function with a capital `R` you might create the impression that is is a Class, not a function.